bindings_ffi/src/mls.rs

```diff
@@ -347,6 +363,21 @@ impl FfiXmtpClient {
         Ok(state.into_iter().map(Into::into).collect())
     }
 
+    /**
+     * Get the inbox_id for each address.
+     */
+    pub async fn inbox_ids_from_addresses(
```

Comment:
I think you can delete this because we already have it here: https://github.com/xmtp/libxmtp/blob/main/bindings_ffi/src/mls.rs#L163-L180